packages/javalib/javalib.2.3.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "sawja@inria.fr"
+build: [
+  ["./configure.sh"]
+  [make "ptrees"]
+  [make "installptrees"]
+  [make]
+]
+bug-reports: "sawja@inria.fr"
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "javalib"]
+  ["ocamlfind" "remove" "ptrees"]
+]
+homepage: "http://sawja.inria.fr"
+depends: [
+  "ocamlfind"
+  "camlzip" {= "1.05"}
+  "camlp4" {>= "4.03"}
```

Comment:
This is an unusual version constraint.  The Camlp4 version is usually tied to the compiler version.  Consequently, this constraint is causing package installation to fail with [most configurations](https://travis-ci.org/ocaml/opam-repository/jobs/147517347), like this:

```
Package is unavailable on this configuration, skipping:
The following dependencies couldn't be met:
  - javalib -> camlp4 >= 4.03
Your request can't be satisfied:
  - camlp4>=4.03 is not available because your system doesn't comply with ocaml-version >= "4.04".
```

In fact from a quick look, it appears that Camlp4 may not be needed at all, although the configure script checks for it.

Could you please consider removing the version constraint, and perhaps removing the `camlp4` dependency altogether?